pyproject.toml

```diff
@@ -1,9 +1,16 @@
 [tool.poetry]
 name = "ferm-docker"
 version = "0.1.0"
-description = "Automatically generate ferm firewall configurations for a local docker instance"
+description = "Automatically generate ferm firewall configuration for a local Docker instance"
 authors = ["Martin Weinelt"]
 license = "GPL-2.0-only"
+readme = "README.rst"
+homepage = "https://github.com/mweinelt/ferm-docker",
+repository = "https://github.com/mweinelt/ferm-docker",
```

Comment:
Syntax highlighting doesn't look too  happy about the commata.

Reply:
Oh indeed, these shouldn't be there :see_no_evil: